internal/config/config.go

```diff
@@ -208,6 +209,34 @@ func (m MCPConfig) ResolvedHeaders() map[string]string {
 	return m.Headers
 }
 
+func resolveCommand(command, logContext string) string {
+	// Handle tilde expansion first
+	if strings.HasPrefix(command, "~/") {
+		homeDir := HomeDir()
+		command = filepath.Join(homeDir, command[2:])
+	} else if command == "~" {
```

Comment:
in what case can the command be just `~` ?

Reply:
fwiw on this home issue there is now a `home` package in crush